app/src/main/java/io/stanwood/bitrise/ui/dashboard/vm/DashboardViewModel.kt

```diff
@@ -84,11 +89,27 @@ class DashboardViewModel(private val router: Router,
         }
     }
 
-    private suspend fun fetchItems() =
+    private suspend fun fetchFavoriteApps(): List<App> =
+        favoriteAppsSlugs
+            ?.map {
+                service
+                    .getApp(token, it)
```

Comment:
We should replace this with the stanwood network library at some point, don't like juggling with tokens here.

Reply:
Does the API also provide a batch request for multiple apps?